src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -204,16 +204,64 @@ <h4 *ngIf="displaySubsection(0, 4)">Setting Up Sessions</h4>
       ></tm-preview-session-panel>
     </tm-example-box>
   </tm-instructor-help-panel>
+  <!-- Question -->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.LET_STUDENT_KNOW_SESSION)"
+                            [id]="SessionsSectionQuestions.LET_STUDENT_KNOW_SESSION"
+                            headerText="How do I let students know about a session?"
+                            [(isPanelExpanded)]="questionsToCollapsed[SessionsSectionQuestions.LET_STUDENT_KNOW_SESSION]">
+    <p>
+      No action is required from you for this. TEAMMATES will send an email (containing submission instructions) to each respondent when the session opens for submissions.
+      After that email has been sent out (you will receive a copy of it, for your reference), you are welcome to inform respondents using another means (outside of TEAMMATES) to look out for the TEAMMATES email.
+    </p>
+    <p>
+      Note that the email generation can take up to 30 minutes from the session start time.
+    </p>
+    <p>
+      <span class="text-danger">Do not use the ‘publish’ feature for this purpose.</span> That feature is for making the responses available for viewing,
+      which you are likely to do after the session deadline is over, unless you want the responses to become visible as they are being collected.
+    </p>
+  </tm-instructor-help-panel>
+  <!-- Question -->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.STUDENT_DID_NOT_RECEIVE_SESSION_EMAIL)"
+                            [id]="SessionsSectionQuestions.STUDENT_DID_NOT_RECEIVE_SESSION_EMAIL"
+                            headerText="What should I do if students say they didn’t receive the TEAMMATES email, as they were supposed to?"
+                            [(isPanelExpanded)]="questionsToCollapsed[SessionsSectionQuestions.STUDENT_DID_NOT_RECEIVE_SESSION_EMAIL]">
+    <p>
+      Here are some remedies:
+    </p>
+    <ol>
+      <li>
+        Use the <button class="btn btn-sm btn-secondary">Remind</button> button on your <b>Home</b> page (there is one for each session) to resend emails to specific students or all those who haven't submitted yet. Also, ask students to check their spam box.
+      </li>
+      <li>
+        Ask students to go to <a href="https://teammatesv4.appspot.com/web/front/help/session-links-recovery">https://teammatesv4.appspot.com/web/front/help/session-links-recovery</a> and request TEAMMATES to resend the links.
```

Comment:
We may be able to use `environment.frontendUrl` here. This will ensure that this link doesn't break if ever we change the frontend url. If this change is desirable, can also use `tmRouterLink` instead of href which is generally preferred for internal links.

Reply:
Should it be something like this: `<a tmRouterLink="/web/front/help/session-links-recovery">{{ frontendUrl + "/web/front/help/session-links-recovery" }}</a>`?
Sorry but I'm not very sure what's a more desired way to retrieve `/web/front/help/session-links-recovery` if there is one.

Reply:
Thank u!